src/System.IO.FileSystem/tests/FileStream/LockUnlock.cs

```diff
@@ -164,7 +164,7 @@ public void OverlappingRegionsFromOtherProcess_ThrowsException(long fileLength,
                         Assert.Throws<IOException>(() => fs2.Lock(long.Parse(secondPos), long.Parse(secondLen)));
                     }
                     return SuccessExitCode;
-                }, path, secondPosition.ToString(), secondLength.ToString()).Dispose();
+                }, $"\"{path}\"", secondPosition.ToString(), secondLength.ToString()).Dispose();
```

Comment:
Not putting the path in parenthesis is actually a bug and the test won't work in environments like mine where there is a space a in the path (see #15891).